### DISCUSSION TITLE
Feasible model is determined to be INFEASIBLE

### DISCUSSION THREAD

**Discussion by Paulyangli** [google/or-tools#3791] [CP-SAT questions]
Hi, I'm using CP-SAT solver v9.4.1874 to build and solve the [model.txt](https://github.com/google/or-tools/files/11504514/model.txt). The model is evidently feasible, but the solver said INFEASIBLE. When I closed constraints one by one, this code was catched up:
```py
    # *** onlyLine constraint
    for k_lanes in lanes_by_onlyLine.values():
        # sat_model.AddAtMostOne([u_i[i] for i in k_lanes])
        sat_model.Add(Sum([u_i[i] for i in k_lanes]) <= 1)
```
What makes me confused is that
1) if I increase 1 to 3 (or above), the optimal solution will be found, but the problem is that the optimal solution completely meet the constraints with the value of 1. BTW,  3 is the size of every k_lanes, I got no idea for this clue.
2) Alternatively, if any coeffs in the Objective is changed to other integer, the model becomes feasible, too. Previously, all coeffs were the same as 6 in the model. BUT, the value (integer) of 'coeffs' is not affecting on the feasiblity of model, isn't it?

Thanks for any help!

**Comment by lperron** [comment]
Check with 9.6

**Comment by Paulyangli** [comment]
Hi Laurent, it works well with v9.6. Thanks very much!
